internal/pkg/db/mongo/export.go

```diff
@@ -67,28 +89,59 @@ func (mc MongoClient) UpdateRegistration(reg export.Registration) error {
 // Get a registration by ID
 // UnexpectedError - problem getting in database
 // NotFound - no registration with the ID was found
-func (mc MongoClient) RegistrationById(id string) (export.Registration, error) {
-	if !bson.IsObjectIdHex(id) {
-		return export.Registration{}, db.ErrInvalidObjectId
+func (mc MongoClient) RegistrationById(id string) (contract.Registration, error) {
+	reg, err := mc.registrationById(id)
```

Comment:
I would inline `registrationById` here and refactor `RegistrationById` as this is its only reference.

Reply:
I'm sort of fond of having the db querying and the conversion to the contract model separated out like this.